### PR TITLE
Add recipe for APScheduler

### PR DESCRIPTION
**Pull request by ddunwoody**

### CODE REVIEW

**Comment by ddunwoody** [comment]
Needs tzlocal - I've just realised my `.condarc` has a few extra channels in it - let me go and see how deep the dependency tree goes!

**Comment by ddunwoody** [comment]
Created #756 with a recipe for tzlocal

**Comment by pelson** [comment]
Now that we have tzlocal, I'll do a close->open cycle to get the CI to run.

File: recipes/apscheduler/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  name: apscheduler
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/agronholm/apscheduler.git
+  git_rev: v{{ version }}
+
+build:
+  number: 0
+  script: python setup.py install
```
Comment (pelson) [inline recipes/apscheduler/meta.yaml]: Sorry, should have spotted it sooner, but whenever using setuptools and conda, please ``python setup.py install --single-version-externally-managed --record=record.txt``.

**Comment by pelson** [comment]
We are green! I'd be happy to merge this with the minor tweak to the build script.

**Comment by ddunwoody** [comment]
@pelson I've added the `setup.py` arguments.  Thanks for merging these in - I have a few others waiting in the wings!

**Pull request merged**